### ISSUE TITLE
No links to PSAT Wiki in the README

### ISSUE THREAD

**Issue by nyb1030** [GoogleChromeLabs/ps-analysis-tool#269]
## Feature Request
No links to PSAT Wiki in the README

PSAT is extremely detailed but hard to follow without clear instructions. There's the [Wiki page](https://github.com/GoogleChromeLabs/ps-analysis-tool/wiki) in the github page, but many people are probably not as familiar to the tab UI, and it is hard to find at the moment. It would be nice to have a link to the wiki at the end of the `Installing and Running PSAT` section of README.

**Comment by amedina** [comment]
Done!
